### ISSUE TITLE
Django Forms integration not working well

### ISSUE THREAD

**Issue by iwalucas** [adamghill/django-unicorn#469]
So I have a regular django form that I am porting to django-unicorn, so I can dynamically update the queryset.

Depending on the approach I have some strange errors.

*Component*
```
<div>
{{form}}
</div>
```

*Unicorn View*
```
from django_unicorn.components import UnicornView,QuerySetType 

class MyView(UnicornView):
    form = None
    class Meta:
        exclude = ()
        javascript_exclude = (
            "form",
        )
    
```

*Django template*
`            {% unicorn 'myview' form=form %}`

Then on any interaction I get:
>Cannot cache component '<class 'project.components.myview.MyView'>' because it is not picklable: <class '_pickle.PicklingError'>: Can't pickle <function paginator_number at 0x1028d6170>: it's not the same object as django.contrib.admin.templatetags.admin_list.paginator_number

I left just the necessary to reproduce the issue....

The form_class documentation is not so clear on how to use that, and when trying, it gives a whole new set of issues.

(awesome project! I have been looking for this for years!)

**Comment by iwalucas** [comment]
Hi @adamghill thanks for the quick response.
Basically I was losing the initial field value when using form_class and accessing my data (update view).... 

I tried to manually populate, but no good:
```
class MyView(UnicornView):
    form_class = MyForm 
    my_cool_field = None

    class Meta:
        exclude = ()
        javascript_exclude = (
            "form",
        )
    
    def mount(self): 
        print(self.form.instance)
        if self.form.instance.id:
            print(self.form.instance.my_cool_field)
            self. my_cool_field = self.form.instance.my_cool_field            

```
Maybe I am doing something wrong? 
The field is a dropdown, and the values are there, but it shows a blank entry

**Comment by iwalucas** [comment]
@adamghill some piece of information. If I remove the model from the unicornview, the value is kept

**Comment by adamghill** [comment]
Interesting! Ok, I'll do some more testing on my own and let you know what I discover/if I find bugs that I can fix.

I assume I could use something like this for `MyForm` for testing purposes?

```python
class MyForm(forms.Form):
    my_cool_field = forms.CharField()
```

**Comment by iwalucas** [comment]
@adamghill yes, if you want to do a zoom or something, I am more than happy to help. Since I am new on django-unicorn, I cant help on fixing things, but I will get there :)

**Comment by elya5** [comment]
I just ran into the same pickling error. I have a GET form that is validated in `views.py`. Then, it gets passed to the UnicornView.

[This stackoverflow answer](https://stackoverflow.com/a/77286987) removed the warning. However, I'm not sure what information gets lost in the process.
